tests/testthat/test-engine-parameters-ranger.R

```diff
@@ -15,6 +15,9 @@ rs <- bootstraps(mtcars, times = 5)
 ## -----------------------------------------------------------------------------
 
 test_that('grid search', {
+  # pending tidymodels/dials#347
+  skip_if_not_installed("dials", minimum_version = "1.3.0.9001")
```

Comment:
See https://github.com/tidymodels/dials/issues/347--currently errors out as dials proposes hyperparameter values that result in failed model fits.